### PR TITLE
Update faq docs for 0.2.0 release

### PR DESCRIPTION
**Pull request by tylerphelan**
https://github.com/pivotal-cf/docs-build-service/issues/56

### CODE REVIEW

File: faq.html.md.erb [hunk 1]
```diff
@@ -22,22 +22,23 @@ By default Build Service will tag each built image twice. The first tag will be
...
 
-* Tanzu Java `io.pivotal.java`
-* Tanzu NodeJS `io.pivotal.nodejs`
-* .NET Core `org.cloudfoundry.dotnet-core`
-* Python `org.cloudfoundry.python`
-* Golang `org.cloudfoundry.go`
-* PHP `org.cloudfoundry.php`
-* HTTPD `org.cloudfoundry.httpd`
-* NGINX `org.cloudfoundry.nginx`
+* Tanzu Java `tanzu-buildpacks/java`
```
Comment (matthewmcnew) [inline faq.html.md.erb:33]: Should we provide links?

File: faq.html.md.erb [hunk 2]
```diff
@@ -22,22 +22,23 @@ By default Build Service will tag each built image twice. The first tag will be
...
+* Paketo NodeJS `paketo-buildpacks/nodejs`
+* Paketo .NET Core `paketo-buildpacks/dotnet-core`
+* Paketo Python `paketo-buildpacks/python`
```
Comment (mgibson1121) [inline faq.html.md.erb:36]: No python :(

**Comment by djoyahoy** [comment]
@tylerphelan  Might be worth having a section for adding a GCR registry secrets. At least one PA struggled to understand that process.

File: faq.html.md.erb [hunk 3]
```diff
@@ -49,72 +50,52 @@ When interacting with a registry or a git repo that has been deployed using a se
...
+	You will be prompted for your github password
 
 ---
 
 ## Why do some builds fail with `Error: could not read run image: *`?
 
 The run image must be publicly readable or readable with the registry credentials configured in a project/namespace.
 
-To see where the build service run image is located run: `pb stack status`.
+To see where the build service run image is located run: `kp stack status STACK-NAME`.
```
Comment (matthewmcnew) [inline faq.html.md.erb:93]: This should have the name of the stack in it.

File: faq.html.md.erb (hunk 1 above)
Comment (tylerphelan) [reply faq.html.md.erb]: https://github.com/pivotal-cf/docs-build-service/pull/62/commits/9be3bae89117aab3693c6e899af3114cc1c38eb9#diff-33e8677d95fd045cfa779061fdd8a063R35

**Comment by tylerphelan** [comment]
Gcr creds for install section: https://github.com/pivotal-cf/docs-build-service/pull/62/commits/9be3bae89117aab3693c6e899af3114cc1c38eb9#diff-33e8677d95fd045cfa779061fdd8a063R110

I think setting the creds for install is the hard part now that the `kp` cli has such a good interface for `gcr` secrets.

**Pull request merged**